pages/dkp/konvoy/2.1/choose-infrastructure/awsgovcloud/clusterconfig/index.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: layout.pug
+navigationTitle: Cluster Configuration
+title: Cluster Configuration
+menuWeight: 10
+excerpt: Create a cluster configuration file and apply it to your environment.
+beta: false
+enterprise: false
+---
+
+## Create the cluster configuration file
+
```

Comment:
Now that we have some breathing room, lets start adding intros to all procedures, even if just to restate the obvious, such as "to create the cluster config file, follow these steps:"